app/components/bulk_upload_summary_component.rb

```diff
@@ -0,0 +1,72 @@
+class BulkUploadSummaryComponent < ViewComponent::Base
+  attr_reader :bulk_upload
+
+  def initialize(bulk_upload:)
+    @bulk_upload = bulk_upload
+    @bulk_upload_errors = bulk_upload.bulk_upload_errors
+    super
+  end
+
+  def upload_status
+    helpers.status_tag(bulk_upload.status, ["app-tag--small govuk-!-font-weight-regular no-max-width"])
+  end
+
+  def setup_errors_count
+    @bulk_upload_errors.where(category: "setup").count
+  end
+
+  def critical_errors_count
+    @bulk_upload_errors.where(category: [nil, "", "not_answered"]).count
```

Comment:
would setup errors be critical too?

Reply:
Aren't setup errors included in important errors?

Reply:
I'd treat everything other than soft-validation as critical. Do setup errors not get displayed in the critical errors section in the report?

Reply:
If you create this as scope if might be a bit more reusable, I left another comment about scopes here https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/2666/files#r1784320513

Reply:
We count setup errors as "errors on important questions" separately from the count for "critical errors". But they do appear within the critical errors section of the error report, but on their own in the summary.